megamek/src/megamek/client/ui/swing/lobby/ChatLounge.java

```diff
@@ -2715,16 +2719,33 @@ public void mouseReleased(MouseEvent e) {
             }
         }
 
-        /** Shows the right-click menu on the mek table */
+        @Override
+        public void mouseMoved(MouseEvent e) {
+            if (popup == null) return;
+           Point p = e.getLocationOnScreen();
+           if (!popup.contains(p))
+           {
+               closePopup();
+           }
+        }
+
+        private void closePopup()
+        {
+            if (popup == null) return;;
+            popup.setVisible(false);
+            popup = null;
+        }
+
+        /** Shows the map selection menu on the map table */
         private void showPopup(MouseEvent e) {
             if (lisBoardsAvailable.isSelectionEmpty()) {
                 return;
             }
             List<String> boards = lisBoardsAvailable.getSelectedValuesList();
             int activeButtons = mapSettings.getMapWidth() * mapSettings.getMapHeight();
             boolean enableRotation = (mapSettings.getBoardWidth() % 2) == 0;
-            ScalingPopup popup = MapListPopup.mapListPopup(boards, activeButtons, this, ChatLounge.this, enableRotation);
-            popup.show(e.getComponent(), e.getX(), e.getY());
+            popup = MapListPopup.mapListPopup(boards, activeButtons, this, ChatLounge.this, enableRotation);
+            popup.show(e.getComponent(), e.getX()-2, e.getY()-2);
```

Comment:
-2? This needs a comment or to not be a magic number.

Reply:
It is also missing spacing.